jobs/rep_windows/spec

```diff
@@ -227,7 +227,13 @@ properties:
   diego.rep.locket.api_location:
     description: "Hostname and port of the locket server"
     default: locket.service.cf.internal:8891
-
+  diego.rep.locket.client_keepalive_time: 
+    description: "Period in seconds after which the locket gRPC client sends keepalive ping requests to the locket server it is connected to."
+    default: 10
+  diego.rep.locket.client_keepalive_timeout: 
+    description: "Timeout in seconds to receive a response to the keepalive ping. If a response is not received within this time, the locket client will reconnect to another server."
+    default: 10
```

Comment:
According to the gRPC doc **keepalive.ClientParameters** Timeout defaults to 20 sec. I doubt that if we overwrite this default to 10 sec. we may introduce backwards incompatible change.

Isn't it possible to set the default Timeout value here to 20 sec. but let the platform operator overtire it.

Reply:
Was not aware that the default was 20s. Will change it right away